airbyte-config/init/src/main/resources/seed/source_definitions.yaml

```diff
@@ -644,7 +644,7 @@
 - name: Paypal Transaction
   sourceDefinitionId: d913b0f2-cc51-4e55-a44c-8ba1697b9239
   dockerRepository: airbyte/source-paypal-transaction
-  dockerImageTag: 0.1.5
+  dockerImageTag: 0.1.6
```

Comment:
We have a script that updates version numbers automatically. This works for now, but don't worry about these in the future - the same goes for the version numbers in source_spec.yaml and the Dockerfile.